shapeshift/web
# fix: use proper yvUSDC vault address

## Description

Use propoer yvUSDC vault address: `0xa354F35829Ae975e850e23e9615b11Da1B3dC4DE`

## Notice

Before submitting a pull request, please make sure you have answered the following:


## Pull Request Type


## Issue (if applicable)

closes #612 

## Testing

1. Visit earn section of app.shapeshift...https://app.shapeshift.com/earn/staking-vaults
2. Notice the APY of USDC
3. Visit Yearn directly: https://yearn.finance/#/vaults
4. Notice the same APY for USDC

## Screenshots (if applicable)

![2021-12-28_15-01](https://user-images.githubusercontent.com/5672954/147575250-4c6dd9e0-2124-4588-bb36-23584a3e3f87.png)
![2021-12-28_15-01_1](https://user-images.githubusercontent.com/5672954/147575258-cfa99817-bc56-4266-9bd4-8ac9beb7563e.png)
